### PR TITLE
Specify reasonable default fonts path

### PR DESCRIPTION
**Pull request by m-kuhn**
Fixes https://github.com/opengisch/QField/pull/4225

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 5363e9c15e34fb8a3059f7bd9fe269d556c3241d: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4228-ed674c4f82ef2e7e155064a7fc1fc50080f97d1d-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4228-ed674c4f82ef2e7e155064a7fc1fc50080f97d1d-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4228-ed674c4f82ef2e7e155064a7fc1fc50080f97d1d-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4228-ed674c4f82ef2e7e155064a7fc1fc50080f97d1d-x86-android.apk)
- [**all access arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4228-ed674c4f82ef2e7e155064a7fc1fc50080f97d1d-arm64-android.apk)
- [all access arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4228-ed674c4f82ef2e7e155064a7fc1fc50080f97d1d-arm-android.apk)
- [all access x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4228-ed674c4f82ef2e7e155064a7fc1fc50080f97d1d-x64-android.apk)
- [all access x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4228-ed674c4f82ef2e7e155064a7fc1fc50080f97d1d-x86-android.apk)
</details>

**Comment by nirvn** [comment]
@m-kuhn , working:
image

File: platform/linux/AppRun [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+HERE=${SELF%/*}
+export PATH="${HERE}/usr/bin/:${HERE}/usr/sbin/:${HERE}/usr/games/:${HERE}/bin/:${HERE}/sbin/${PATH:+:$PATH}"
+export LD_LIBRARY_PATH="${HERE}/usr/lib/:${HERE}/usr/lib/i386-linux-gnu/:${HERE}/usr/lib/x86_64-linux-gnu/:${HERE}/usr/lib32/:${HERE}/usr/lib64/:${HERE}/lib/:${HERE}/lib/i386-linux-gnu/:${HERE}/lib/x86_64-linux-gnu/:${HERE}/lib32/:${HERE}/lib64/${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"
+export PYTHONPATH="${HERE}/usr/share/pyshared/${PYTHONPATH:+:$PYTHONPATH}"
+export XDG_DATA_DIRS="${HERE}/usr/share/${XDG_DATA_DIRS:+:$XDG_DATA_DIRS}"
+export PERLLIB="${HERE}/usr/share/perl5/:${HERE}/usr/lib/perl5/${PERLLIB:+:$PERLLIB}"
+export GSETTINGS_SCHEMA_DIR="${HERE}/usr/share/glib-2.0/schemas/${GSETTINGS_SCHEMA_DIR:+:$GSETTINGS_SCHEMA_DIR}"
+export QT_PLUGIN_PATH="${HERE}/usr/lib/qt4/plugins/:${HERE}/usr/lib/i386-linux-gnu/qt4/plugins/:${HERE}/usr/lib/x86_64-linux-gnu/qt4/plugins/:${HERE}/usr/lib32/qt4/plugins/:${HERE}/usr/lib64/qt4/plugins/:${HERE}/usr/lib/qt5/plugins/:${HERE}/usr/lib/i386-linux-gnu/qt5/plugins/:${HERE}/usr/lib/x86_64-linux-gnu/qt5/plugins/:${HERE}/usr/lib32/qt5/plugins/:${HERE}/usr/lib64/qt5/plugins/${QT_PLUGIN_PATH:+:$QT_PLUGIN_PATH}"
```
Comment (nirvn) [inline platform/linux/AppRun:10]: @m-kuhn , are we sure we want Qt4 plugins paths in there?

Comment (m-kuhn) [reply platform/linux/AppRun:10]: :fearful: thanks for noticing, I was copying this from somewhere in the internet

**Comment by m-kuhn** [comment]
@kannes you tested the result of this pull request in https://github.com/opengisch/QField/issues/4225#issuecomment-1537125850, correct? Sorry, I have to trust your results because I cannot reproduce it on my machine.

**Comment by kannes** [comment]
Yes! I added a screenshot now. The artifact does not have the issue anymore, the fonts look good.

**Review by nirvn — APPROVED** [review]

**Pull request merged**